conans/client/conan_command_output.py

```diff
@@ -30,9 +30,21 @@ def profile_list(self, profiles):
     def remote_list(self, remotes, raw):
         for r in remotes:
             if raw:
-                self._output.info("%s %s %s" % (r.name, r.url, r.verify_ssl))
+                if r.disabled:
+                    self._output.info(
+                        "%s %s %s %s" %
+                        (r.name, r.url, r.verify_ssl, r.disabled))
```

Comment:
This changes current `raw` output adding another item at the end, it could be considered breaking... the only way to keep previous output is to ignore disabled remotes in the `raw` list 🤔 What do you think about this issue?